src/ch/ch_migration.c

```diff
@@ -306,7 +307,23 @@ chDomainMigrationSrcPerform(virCHDriver *driver,
     (void) dname;
     (void) flags;
 
-    return -1;
+    VIR_DEBUG("t-kasanchez %s", uri_str);
+    cmd = virCommandNew("socat");
+    virCommandAddArg(cmd, "UNIX-LISTEN:/var/run/libvirt/ch/ch_impish_nfs-migr-send, reuseaddr");
+    virCommandAddArg(cmd, uri_str);
+    if (virCommandRunAsync(cmd, NULL) < 0)
+        return -1;
+    
+    cmd = virCommandNew("ch-remote");
+    virCommandAddArgPair(cmd, "--api-socket", "/var/run/libvirt/ch/ch_impish_nfs-socket");
```

Comment:
Could you follow virCHMonitorNew and derive the socket paths, instead of hardcoding?